config/config.go

```diff
@@ -86,6 +91,15 @@ type (
 		MaxHeaderBytes    int           `yaml:"maxHeaderBytes"`
 	}
 
+	MembershipConfig struct {
+		// total shard count across all the async servers
+		Shard int `yaml:"shard"`
+		// the advertise address to bind
```

Comment:
It's better to have a different `BindAddress` and default to `AdvertiseAddress`. 

In production env this could be different for networking setup(security etc). User may just bind for 0.0.0.0:1234 for binding but there is a network router to use node1:1234 as the real address

Reply:
How are you going to implement the network router?

Reply:
Done.